Editing the installing CLI (#933)
fishtown-analytics/docs.getdbt.com: website/docs/dbt-cli/configure-your-profile.md

---
title: "Configuring your profile"

> description: "You can configure your profile using your computer's command line."

Author: ```suggestion
description: "Configure your profile using the command line."
```

---

## Related documentation

> * [`profiles.yml` reference](/reference/profiles.yml): Learn more about profile configuration.

Author: What do you think about identifying "related docs" so people can understand what they will be clicking?

Reviewer: yes absolutely!
